redCOMPONENT-COM/redSHOP
# REDSHOP-1559 XTREM-57 #comment Rewrite EpayV2 to fix Capture, Refund and Cancel issues

https://redweb.atlassian.net/browse/XTREM-57
# Testing Info
- Use epay sandbox detail from redSHOP wiki 
- Try everything, place order, cancel order, shipped order. Everything should work with or without ideal errors.
